client/src/app/pages/applications/components/application-detail-drawer/application-detail-drawer.tsx

```diff
@@ -303,6 +308,18 @@ const TabDetailsContent: React.FC<{
         onEditClick={onEditClick}
         onCloseClick={onCloseClick}
       />
+      <TextContent className={spacing.mtLg}>
```

Comment:
Since this is talking about the targets for an analysis, this content should be on the Reports tab.

Reply:
@sjd78, @rszwajko, the information was added to the app details tab according to this issue: #2012 
Could you please confirm where it should be added? Thanks